st2common/tests/unit/services/test_workflow.py

```diff
@@ -32,8 +32,10 @@
 from st2common.exceptions import action as action_exc
 from st2common.models.db import liveaction as lv_db_models
 from st2common.models.db import execution as ex_db_models
+from st2common.models.db.pack import ConfigDB
 from st2common.persistence import execution as ex_db_access
 from st2common.persistence import workflow as wf_db_access
+from st2common.persistence.pack import Config
```

Comment:
Since the pack module is imported above, this import is not required.

Reply:
Have to have it in order to save it to DB. Later when line 569 `param_utils.render_live_params` is called, it read config information from DB.

Reply:
Ok. I see.  I misread.  You have to change the import here to `from st2common.persistence import pack as pk_db_access`.

Reply:
Yes.